test_tf2/test/test_buffer_client.cpp

```diff
@@ -36,21 +36,32 @@
 *********************************************************************/
 #include <gtest/gtest.h>
 #include <tf2_ros/buffer_client.h>
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 #include <tf2_geometry_msgs/tf2_geometry_msgs.h>
 #include <tf2_kdl/tf2_kdl.h>
 #include <tf2_bullet/tf2_bullet.h>
+#include <chrono>
+#include <thread>
```

Comment:
`#include <functional>` for `std::bind`, `#include <memory>` for `std::make_unique`.